recipe/meta.yaml

```diff
@@ -9,11 +9,28 @@ source:
   fn: vigra-{{ version_tag }}-src.tar.gz
   url: https://github.com/ukoethe/vigra/archive/{{ version_tag }}.tar.gz
   sha1: 518c7a9ef0e26e39563aeebf22bcee55a7d257c8
+  patches:
+    ####################################################
+    # Workaround for VS 2015                           #
+    #                                                  #
+    # xref: https://github.com/ukoethe/vigra/pull/412  #
+    ####################################################
+    - PR_412.diff
+
+    ####################################################
+    # Fix Windows Python test run                      #
+    #                                                  #
+    # xref: https://github.com/ukoethe/vigra/pull/416  #
+    ####################################################
+    - PR_416.diff
 
 build:
-  skip: true  # [win]
-  number: 0
+  skip: true  # [win and py<35]
+  number: 1
   detect_binary_files_with_prefix: true
+  features:
+    - vc14               # [win and py35]
+    - vc14               # [win and py36]
```

Comment:
Should add that we don't need these features. They are only needed for non-Python libraries. When it comes to Python libraries, simply depending on Python is enough to ensure the correct runtime is used.